src/app/helptext/task-calendar/replication/replication-wizard.ts

```diff
@@ -87,6 +87,9 @@ export default {
  <i>Destination Dataset</i>. Select a preset schedule or choose \
  <i>Custom</i> to use the advanced scheduler.'),
 
+    readonly_placeholder: T('Set Destination Dataset Read-only'),
+    readonly_tooltip: T(''),
```

Comment:
Suggested help text:

Setting this option changes the destination dataset to be read-only. To continue using the default or existing dataset read permissions, leave this option unset.